index.html

```diff
@@ -50,6 +50,7 @@
 <body>				
 	<nav>
 		<div class="language-menu" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
+			<span class="newsletter-date">July-October, 2019</span>
```

Comment:
I don't know what happened, but in my machine looks well

![language-menu](https://user-images.githubusercontent.com/29107510/78844374-8da9d780-79d3-11ea-8aaa-353b118a73b4.png)

Reply:
>     2\. We probably need to update our schema. We may want to consider another schema because now we have a date. But for now, take a look at creative work to determine how to properly mark up the date

Vamos a usar datePublished y copyrightYear. Gracias por buscar esa info